src/sentry/static/sentry/app/views/organizationIntegrations/integrationDetailedView.tsx

```diff
@@ -0,0 +1,420 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import keyBy from 'lodash/keyBy';
+import {RouteComponentProps} from 'react-router/lib/Router';
+
+import {Organization, Integration, IntegrationProvider} from 'app/types';
+import {RequestOptions} from 'app/api';
+import {addErrorMessage} from 'app/actionCreators/indicator';
+import {Hooks} from 'app/types/hooks';
+import {t} from 'app/locale';
+import {trackIntegrationEvent} from 'app/utils/integrationUtil';
+import AsyncComponent from 'app/components/asyncComponent';
+import PluginIcon from 'app/plugins/components/pluginIcon';
+import space from 'app/styles/space';
+import AddIntegrationButton from 'app/views/organizationIntegrations/addIntegrationButton';
+import Access from 'app/components/acl/access';
+import Tag from 'app/views/settings/components/tag';
+import Button from 'app/components/button';
+import Alert from 'app/components/alert';
+import Tooltip from 'app/components/tooltip';
+import InlineSvg from 'app/components/inlineSvg';
+import ExternalLink from 'app/components/links/externalLink';
+import InstalledIntegration, {
+  Props as InstalledIntegrationProps,
+} from 'app/views/organizationIntegrations/installedIntegration';
+import marked, {singleLineRenderer} from 'app/utils/marked';
+import HookStore from 'app/stores/hookStore';
+import withOrganization from 'app/utils/withOrganization';
+import {growDown, highlight} from 'app/styles/animations';
+import {sortArray} from 'app/utils';
+
+type State = {
+  configurations: Integration[];
+  information: {providers: IntegrationProvider[]};
+  tab: string;
+  newlyInstalledIntegrationId: string;
+};
+
+type Props = {
+  organization: Organization;
+} & RouteComponentProps<{orgId: string; providerKey: string}, {}>;
+
+const defaultFeatureGateComponents = {
+  IntegrationFeatures: p =>
+    p.children({
+      disabled: false,
+      disabledReason: null,
+      ungatedFeatures: p.features,
+      gatedFeatureGroups: [],
+    }),
+  FeatureList: p => {
+    return (
+      <ul>
+        {p.features.map((f, i) => (
+          <li key={i}>{f.description}</li>
+        ))}
+      </ul>
+    );
+  },
+} as ReturnType<Hooks['integrations:feature-gates']>;
+
+const tabs = ['information', 'configurations'];
+
+class IntegrationDetailedView extends AsyncComponent<
+  Props & AsyncComponent['props'],
+  State & AsyncComponent['state']
+> {
+  componentDidMount() {
+    const {location} = this.props;
+    const value =
+      typeof location.query.tab === 'string' ? location.query.tab : 'information';
+    this.setState({
```

Comment:
@NisanthanNanthakumar you should add the eslint ignore comment for this so we can run the pre-commit hooks